packages/eslint-config-nimble-react/package.json

```diff
@@ -22,7 +22,8 @@
     "lib"
   ],
   "dependencies": {
-    "@nimblehq/eslint-config-nimble-core": "^2.7.0",
+    "@nimblehq/eslint-config-nimble-core": "^2.8.0",
+    "@nimblehq/eslint-config-nimble-typescript": "^1.0.0",
```

Comment:
```suggestion
    "@nimblehq/eslint-config-nimble-core": "2.8.0",
    "@nimblehq/eslint-config-nimble-typescript": "1.0.0",
```
Same for others 👍

Reply:
I think we can keep it to allow this version or the above one, what do you think? 🤔

Reply:
I'm fine with either, but I think it's better to keep them up-to-date at once, otherwise, we would upgrade 1 package and forget on others 🤔

Reply:
It makes total sense to use the precise version, It was updated in efd71ee ✅ 🙏🏻